ext/fonts/spleen_8x16.c

```diff
@@ -0,0 +1,223 @@
+/*
```

Comment:
Why you can not use existing fonts in `subsys/fb/cfb_fonts.c ` or generate a new subset with `./scripts/gen_cfb_font_header.py` ? See also my comment [there](https://github.com/zephyrproject-rtos/zephyr/pull/16331#pullrequestreview-245969867).

Reply:
I've removed this font.